cmd/kubectl-kubtest/commands/install.go

```diff
@@ -31,13 +49,54 @@ func NewInstallCmd() *cobra.Command {
 			out, err := process.Execute("helm", "upgrade", "--install", "--namespace", namespace, name, chart)
 			ui.ExitOnError("executing helm install", err)
 
-			ui.Info("Helm output", string(out))
+			ui.Info("Helm install output", string(out))
+			err = printDashboardAddress(k8sClient, namespace)
+			ui.PrintOnError("getting dashboard address", err)
 		},
 	}
 
 	cmd.Flags().StringVar(&chart, "chart", "kubeshop/kubtest", "chart name")
 	cmd.Flags().StringVar(&name, "name", "kubtest", "installation name")
 	cmd.Flags().StringVar(&namespace, "namespace", "default", "namespace where to install")
-
+	cmd.Flags().BoolVarP(&installIngress, "ingress", "i", false, "install ingress if not present in the cluster to expose the endpoint for the dashboard")
 	return cmd
 }
+
+func installIngressController(k8sClient *kubernetes.Clientset, namespace string) error {
```

Comment:
We know this will work only on gcloud, I tried setting it up on bare metal and it didn't work. 
Maybe it is worth mentioning it somewhere in the docs.

Reply:
Done